samples/characterization/gateset-tomography/gateset-tomography.ipynb

```diff
@@ -0,0 +1,1850 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0fa6fef9",
+   "metadata": {},
+   "source": [
+    "# Gateset Tomography with Python and Q# #"
```

Comment:
```suggestion
    "# Gateset Tomography with Python and Q#"
```
I think the recommendation for markdown headers is to only have the hash symbol in the beginning of the line.

Reply:
Agreed, this was a bit of a hack, since the heading ends in a "#." The problem is not all clients support escaping the hash in Q#, so this is the safest way to typeset "Q#" at the end of a header...

Reply:
True, that makes sense. Then we better leave it there :)